src/lib/components/layout/Toolbar/__tests__/ToolbarSpacer.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ToolbarSpacer from '../ToolbarSpacer';
+
+describe('rendering', () => {
+  it('renders correctly with a single child', () => {
```

Comment:
I think the description should read: `renders correctly`
There are no children involved.